src/Web/Slack/Experimental/Blocks.hs

```diff
@@ -163,22 +163,43 @@ truncateSlackMessage (SlackMessage blocks) =
    in (SlackMessage truncatedBlocks, or isTruncateds)
 
 truncateSlackBlock :: SlackBlock -> (SlackBlock, Bool)
-truncateSlackBlock sb@(SlackBlockSection (SlackText texts) mAccessory) =
-  let messageLength = sum $ map T.length texts
+truncateSlackBlock sb@(SlackBlockSection SlackSection {..}) =
```

Comment:
The entire premise of this function is bizarre and I think it should be removed from the library. Userland should be responsible for handling messages that exceed the length limit. 

For that reason, I am not taking the `fields` text into account here. It would be more trouble than it's worth, imo.